src/frontend/data/stores/useJitsiApi/index.ts

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'utils/createContext';
+import { Maybe } from 'utils/types';
+
+const store = createStore<Maybe<JitsiMeetExternalAPI>>('JitsiApiProvider');
+
+export const JitsiApiProvider = store.Provider;
+export const useJitsiApi = store.useStore;
```

Comment:
I'm not sure to understand what you expect to test here.

Reply:
I just mean that the majority of others stores are tested
```
├── useChatItemsStore
│   ├── index.spec.tsx
│   └── index.ts
├── useCurrentUser
│   └── index.tsx
├── useDocument
│   ├── index.spec.tsx
│   └── index.ts
├── useLiveFeedback
│   └── index.ts
├── useLiveModale
│   └── index.ts
├── useLivePanelState
│   ├── index.spec.ts
│   └── index.ts
├── useLiveSession
│   ├── index.spec.ts
│   └── index.ts
├── useLiveStateStarted
│   ├── index.spec.tsx
│   └── index.ts
├── useObjectProgress
│   ├── index.spec.tsx
│   └── index.tsx
├── useParticipantsStore
│   ├── index.spec.tsx
│   └── index.ts
├── useParticipantWorkflow
│   ├── index.spec.ts
│   └── index.ts
├── useSetDisplayName
│   └── index.tsx
├── useThumbnail
│   ├── index.spec.tsx
│   └── index.tsx
├── useTimedTextTrack
│   ├── index.spec.tsx
│   └── index.tsx
├── useTimedTextTrackLanguageChoices
│   ├── index.spec.tsx
│   └── index.tsx
├── useTranscriptTimeSelector
│   └── index.tsx
├── useVideo
│   ├── index.spec.tsx
│   └── index.tsx
└── useVideoProgress
    └── index.tsx
    ```

Reply:
Yes because those stores are zustand stores with custom exposed state methods, store based on createStore are not tested since there is not much to test (calling a method do init a global var and exposing 2 elements in the file).
Eventually we may test createStore even if it wont change much imo

Reply:
IMO there is no test to write here.